packages/utils/src/lib/report.ts

```diff
@@ -161,6 +166,105 @@ export function countCategoryAudits(
   }, 0);
 }
 
+export function getAuditsFromAllPlugins(
+  report: ScoredReport,
+): EnrichedAuditReport[] {
+  return report.plugins.reduce<EnrichedAuditReport[]>(
+    (acc, plugin) => [...acc, ...plugin.audits],
+    [],
+  );
```

Comment:
You can use `flatMap` here:

```suggestion
  return report.plugins.flatMap(plugin => plugin.audits);
```

Reply:
Removed that function - as it was no more used due to the code optimization